requirements-docs.txt

```diff
@@ -0,0 +1,6 @@
+git+https://github.com/SamuelMarks/griffe.git@rest
```

Comment:
What is this dependency? Do we need to take it from a personal repo?

Reply:
Oh this can be removed now; it was a fork I made to support use my cdd-python to parse docstrings and more [argparse, functions, classes, etc.]